multinet/uploaders/swagger/csv.yaml

```diff
@@ -6,7 +6,8 @@ consumes:
 parameters:
   - $ref: "#/parameters/workspace"
   - $ref: "#/parameters/table"
-  - name: data
+  -
+    name: data
```

Comment:
Why break the line? YAML lets you create an object without this extra newline.

Reply:
I just thought it was easier to read, and I'm not sure if there's a standard for this, but every yaml linter/validator that I've used has formatted it to be this way, so I thought it was preferred.

Reply:
Huh if the linters like it better then I suppose we should keep it.

Maybe we should have a linter/validator for these files...

Reply:
There is [YamlLint](https://github.com/adrienverge/yamllint), which could be useful. If we're serious about this we should open an issue for it.

Reply:
It's pretty low-priority but yeah we should do it. This is much more useful if there's also a validator for swagger YAML files.

Reply:
@AlmightyYakob, did you open an issue for this?

Reply:
I did not, I'll do it now

Reply:
#353